data-model/SDP-data-model-templates/place_sdp-template.json

```diff
@@ -13,9 +23,26 @@
       "uniform_name": ""
     }
   ],
-  "associated_entities": [
+  "related_persons": [
+    {
+      "relation": "CV",
+      "id": "URI/ARK",
+      "note": [
+        "array of strings"
+      ]
+    }
+  ],
+  "related_places": [
+    {
+      "relation": "CV",
+      "id": "URI/ARK",
+      "note": [
+        "array of strings"
+      ]
+    }
+  ],
+  "related_works": [
```

Comment:
This would include place of composition relationships (for the _work_, not a particular object). This may not be worth including. I added it only for the sake of being comprehensive with authority to authority relation options.